### PR TITLE
US78213 - use the correct image class so that the focal point isn't cropped out of the course tile

### PR DESCRIPTION
**Pull request by jstefaniuk-d2l**

### CODE REVIEW

File: d2l-utility-behavior.html [hunk 1]
```diff
@@ -44,8 +44,8 @@
 		* @param {Entity} image
 		* @return {String}
 		*/
-		getDefaultTileImageLink: function(image) {
```
Comment (CodeBaboon) [inline d2l-utility-behavior.html]: Maybe rename to `getDefaultImageLink()`, since it could be a tile or a banner now?

**Comment by CodeBaboon** [comment]
LGTM

**Comment by jstefaniuk-d2l** [comment]
Anyone want to do a quick test pass?

**Comment by trevorjonesd2l** [comment]
I'll take a peak

**Comment by ryantmer** [comment]
:mountain: ?

**Comment by trevorjonesd2l** [comment]
- [ ] Just mentioned to this to Jeff but I'll put it here too to keep track of. Hover state/animation is off a bit on Firefox - image slides to the left leaving a blue bar on the right.

image

*meow*

**Comment by trevorjonesd2l** [comment]
Looks good 👍

**Pull request merged**